src/const.py

```diff
@@ -14,11 +14,17 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
-SERVICE_NAME = 'rgw_setup'
+from enum import Enum
+
+COMPONENT_NAME = 'rgw'
+SERVICE_NAME = f'{COMPONENT_NAME}_setup' # rgw_setup
 INSTALL_PATH = '/opt/seagate/cortx'
-RGW_INSTALL_PATH = f'{INSTALL_PATH}/rgw'
-RGW_CONF_TMPL = f'{RGW_INSTALL_PATH}/conf/cortx_rgw.conf'
-RGW_CONF_FILE = 'cortx_rgw.conf'
+RGW_INSTALL_PATH = f'{INSTALL_PATH}/{COMPONENT_NAME}'
```

Comment:
value is substituted nicely, which matters functionally. though var name everywhere has RGW, can be COMP_INSTALL_PATH, etc. but thats fine, we can target that in any future commit